client/my-sites/plugins/use-plugins/index.ts

```diff
@@ -1,7 +1,7 @@
 import { isEnabled } from '@automattic/calypso-config';
 import { useMemo } from 'react';
 import { Plugin } from 'calypso/data/marketplace/types';
-import { useSiteSearchPlugins } from 'calypso/data/marketplace/use-site-search-es-query';
+import { useNewSiteSearchPluginsInfinite } from 'calypso/data/marketplace/use-site-search-es-query';
```

Comment:
Do we also have an `useOldSiteSearchPluginsInfinite`? if not can we omit the `New`? Also, `Site Search` is the name of the product we are using which uses elastic search. Would it be simpler renaming to `use-es-query` and `useESPluginsInfinite`?